en-US/snippets/2017/apr2017.lang

```diff
@@ -0,0 +1,41 @@
+## NOTE: Reference bug 1345221; Deadline: 21 March, 2017
+## NOTE: Guidelines on localizing snippets: https://goo.gl/PluA97.  The tone is QUAC (Quick, Untamed, Alert, Clever). Do your best to make it local and culturally relevant.
+## NOTE: Campaign: cross platform promotion for Firefox Focus through snippet and email
+## NOTE: There is no character length limit
+
+
+# Campaign: Firefox desktop snippet
+;Block unwanted ads and trackers by default with the new Firefox Focus for Android.  Get it now.
+Block unwanted ads and trackers by default with the new Firefox Focus for Android.  Get it now.
+
+
+# Campaign: Email subject line for Firefox Focus on iOS and Android
+;Your Firefox Focus download link
+Your Firefox Focus download link
+
+
+# Campaign: Email headline for Firefox Focus on iOS and Android
+;Seriously easy privacy.
+Seriously easy privacy.
+
+
+# Campaign: Email body text for Firefox Focus on iOS and Android
+# URL for iOS: https://itunes.apple.com/app/firefox-focus-the-privacy-browser/id1055677337?mt=8
+# No link for Android
+;Block unwanted ads and trackers by default with Firefox Focus. Install it today.
+Block unwanted ads and trackers by default with Firefox Focus. Install it today.
```

Comment:
Do we know if the entire string is going to become a link or just the last part? If it's the latter, we should mark it with `<a>Install it today.</a>`

Why no link for Android? Do you have more info besides the spreadsheet?